.gitignore

```diff
@@ -0,0 +1 @@
+test.sh
```

Comment:
this is not included in your commit

Reply:
I did not commit it intentionally. It's a very short script that sets some environment vars and runs script.sh. It allows me to test on our org but contains a token so I did not commit it. I removed from the branch.